projects/canopy/src/lib/card/docs/card.stories.ts

```diff
@@ -179,9 +179,12 @@ export default {
 const defaultCardTemplate = `
 <lg-card>
   <lg-card-header>
-    <lg-card-title [headingLevel]="headingLevel">
-      {{title}}
-    </lg-card-title>
+    <lg-card-navigation-title *ngIf="link else noLink" [title]="title" [headingLevel]="headingLevel"></lg-card-navigation-title>
```

Comment:
Because you have a specific story for the `navigationCardTemplate` now I think we can keep the `defaultCardTemplate` simple with only the `lg-card-title` component.